final_challenges/parking_violations.js

```diff
@@ -3,5 +3,13 @@ var request = require('request');
 // You can find information about NYC parking violations at "https://data.cityofnewyork.us/resource/dbw3-ymb4.json"
 
 // Below, write code that prints out the definition for all parking violations with a fine more than $110 below 96th street in Manhattan.
-
+request("https://data.cityofnewyork.us/resource/dbw3-ymb4.json#"), 
+function(err, res, body)
+var res = JSON.parse(body);
+var parking  = response.definitionList;
+var parkingviolationName = []
+for (var i = 0; i < parking.length; i++){
+    parkingviolationName.push(parking[i].parkingviolationName);
+}
+console.log(parkingviolationName)
```

Comment:
You don't want to commit changes to this file - just the one in your folder. This file should remain a blank exercise. Please remove it from this PR.